README.md

```diff
@@ -119,7 +144,7 @@ stringit, frustra Saturnius uteroque inter! Oculis non ritibus Telethusa
 ```
 
 ## Page Header
-The first heading should be # Head with `---` like below and should refer to the overall title of the document. The right nav **only** starts on the second level of headings. 
+The first heading should be # Head with `---` like below and should refer to the overall title of the document.
```

Comment:
i really really want to fix this. The first heading should just be a `#` and the horizontal line should be added in with css not a hr element.

Reply:
given that the "first" header will be re-written to be a different heading level depending on its depth in the tree, it doesn't even make sense to include explict hr elements, as they will be inconsistently applied to either `h1`s `h2`s or `h3`s depending on what level this doc starts at

Reply:
yes yes!! https://github.com/filecoin-project/specs/issues/1081